robottelo/test.py

```diff
@@ -18,8 +18,11 @@
 from datetime import datetime
 from fabric.api import execute, settings
 from robottelo.cli.metatest import MetaCLITest
+from robottelo.cli.org import Org as OrgCli
```

Comment:
Please don't import using the `as` keyword unless there is a *very* good reason.